examples/Makefile

```diff
@@ -27,19 +27,31 @@
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ###############################################################################
 
-
 EXAMPLES := adder/tests \
-	    endian_swapper/tests \
             axi_lite_slave/tests \
+            dff/tests \
+            endian_swapper/tests \
+            mean/tests \
+            mixed_language/tests \
+            ping_tun_tap/tests
```

Comment:
This last test requires root permissions to run (and that's why it's failing currently in Travis). Requiring root to run "make test" isn't reasonable. Please remove this test from the list. You can optionally run the test by itself using sudo in Travis.

Reply:
Cleaned up in #1135.